### PR TITLE
Dac6 1009

### PR DESCRIPTION
**Pull request by mbiczak**

### CODE REVIEW

**Review by ers2015112 — COMMENTED** [review]
Do we need Navigation tests as part of this ticket as at the moment they aren't any?

File: app/controllers/CheckYourAnswersController.scala [hunk 1]
```diff
@@ -17,16 +17,21 @@
 package controllers
 
 import controllers.actions.{DataRequiredAction, DataRetrievalAction, IdentifierAction}
+import models.{NormalMode, UserAnswers}
```
Comment (iservport) [inline app/controllers/CheckYourAnswersController.scala:20]: Some imports are not required if you remove the unnecessary construction arguments.

````
import models.{NormalMode, UserAnswers}
import org.slf4j.LoggerFactory
import pages.ContactNamePage
````

File: app/utils/RowBuilder.scala [hunk 2]
```diff
@@ -41,12 +41,12 @@ trait RowBuilder {
       .msg()
       .withArgs(((if (isPlural) argIfPlural else argIfSingular) +: msgKey.tail): _*)
 
-  private[utils] def toRow(msgKey: String, content: Content, href: String, columnWidth: String = "govuk-!-width-one-half")(implicit messages: Messages): Row = {
+  private[utils] def toRow(msgKey: String, value: Content, href: String, columnWidth: String = "govuk-!-width-one-half")(implicit messages: Messages): Row = {
```
Comment (iservport) [inline app/utils/RowBuilder.scala:44]: 👍

File: test/controllers/CheckYourAnswersControllerSpec.scala [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+
+      val json                = jsonCaptor.getValue
+      val firstContactDetails = (json \ "firstContactList").toString
+
+      templateCaptor.getValue mustEqual "checkYourAnswers.njk"
+      firstContactDetails.contains("Contact name") mustBe true
+      firstContactDetails.contains("Email address") mustBe true
+      firstContactDetails.contains("Telephone number") mustBe isFirstContactPhone
+
+      application.stop()
```
Comment (iservport) [inline test/controllers/CheckYourAnswersControllerSpec.scala:103]: Following the above, this would be not necessary.

**Review by iservport — APPROVED** [review]
LGTM

**Pull request merged**